src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -27,6 +33,20 @@ const consultationRoutes: AppRoutes = {
         patientId={patientId}
       />
     ),
+  "/facility/:facilityId/patient/:patientId/encounter/:encounterId/drawings": ({
+    encounterId,
+  }) => (
+    <Suspense fallback={<Loading />}>
+      <ExcalidrawPage associatingId={encounterId} fileType="encounter" />
+    </Suspense>
+  ),
+  "/facility/:facilityId/patient/:patientId/encounter/:encounterId/drawings/:drawingId":
+    ({ drawingId }) => (
+      <Suspense fallback={<Loading />}>
+        <ExcalidrawView drawingId={drawingId} />
+      </Suspense>
+    ),
```

Comment:
@bodhish marking this for hold since backend would not allow editing a file due to audit reasons.

@vigneshhari suggests making a plug and writing the elements to the DB itself instead of a file upload.

Reply:
Buckets have versions what stops us from editing 🤔